lib/web_ui/lib/src/engine/platform_views/message_handler.dart

```diff
@@ -73,8 +73,10 @@ class PlatformViewMessageHandler {
     if (!_contentManager.knowsViewType(viewType)) {
       callback(_codec.encodeErrorEnvelope(
         code: 'unregistered_view_type',
-        message: 'trying to create a view with an unregistered type',
-        details: 'unregistered view type: $viewType',
+        message: 'A HtmlElementView widget is trying to create a platform view'
+            ' with an unregistered type: <$viewType>.',
```

Comment:
Just to make the spacing consistent.
```suggestion
        message: 'A HtmlElementView widget is trying to create a platform view '
            'with an unregistered type: <$viewType>.',
```